Assignments/Assignment 7/Assignment_7.tex

```diff
@@ -194,12 +194,12 @@
 	Therefore,
 	\begin{align*}
 		\langle f + h , g \rangle\ & = \left( f(-\pi) + h(-\pi) \right) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( f''(x) + h''(x) \right) g''(x) \dif x                                                    \\
-                                           & = f(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( f''(x) + g''(x) \right) \dif x + h(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( h''(x) + g''(x) \right) \dif x \\
+                                           & = f(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( f''(x)  g''(x) \right) \dif x + h(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( h''(x)  g''(x) \right) \dif x \\
                                            & = \langle f,g \rangle + \langle h,g \rangle
 	\end{align*}
 	Therefore,
 	\begin{align*}
-		\langle \alpha f , g \rangle & = \alpha f(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( \alpha f''(x) + g''(x) \right) \dif x \\
+		\langle \alpha f , g \rangle & = \alpha f(-\pi) g(-\pi) + \int\limits_{-\pi}^{\pi} \left( \alpha f''(x)  g''(x) \right) \dif x \\
```

Comment:
You didn't remove the space after the '+', so now there are 2 spaces between the two terms. 😠